perf_benchmark/benchmark_input_stream.cc

```diff
@@ -0,0 +1,160 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+//
+
+#include "perf_benchmark/benchmark_input_stream.h"
+
+namespace google {
+namespace grpc {
+namespace transcoding {
+
+namespace perf_benchmark {
+
+BenchmarkZeroCopyInputStream::BenchmarkZeroCopyInputStream(std::string msg,
+                                                           uint64_t chunk_per_msg)
```

Comment:
should this be `num_chunks_per_msg`?  Kinda confused by naming

Reply:
Yes I agree. Sorry about the confusion!